bridges/snowbridge/pallets/system-frontend/src/lib.rs

```diff
@@ -94,6 +94,9 @@ pub mod pallet {
 		/// Universal location of this runtime.
 		type UniversalLocation: Get<InteriorLocation>;
 
+		/// The global flag when set to true, all non-governance operations are disabled.
+		type PauseFlag: Get<bool>;
```

Comment:
I think the storage for the paused flag should be within the pallet, so that we can create privileged extrinsics to (un)pause.

Otherwise governance will need to use raw storage keys in upgrades when (un)pausing, which is bad.

You make this pallet implement an interface like `Pausable`  so that other parts of the runtime (the exporter for example) can check paused status.

Reply:
https://github.com/yrong/polkadot-sdk/pull/12/commits/4707a124e212f84d4afba34bea215cca86fd494f